rapidjson/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+if (BUILDING_LEATHERMAN)
+    set(include_dir "${PROJECT_SOURCE_DIR}/vendor/rapidjson/include")
+    set(${include_var} ${include_dir} PARENT_SCOPE)
+
+    if(LEATHERMAN_INSTALL)
+        install(DIRECTORY "${include_dir}" DESTINATION "include/leatherman/vendor/rapidjson")
```

Comment:
I think this will need to copy the `rapidjson` directory in `${include_dir}`. The `${include_var}` at line 9 will add the destination to include paths, and `install(DIRECTORY ...)` will move the named directory to the destination.

So the way this is currently written, I'd expect to find `include/leatherman/vendor/rapidjson/include/rapidjson/*`, which would require doing `include <include/rapidjson/*>` to include something. We don't have good tests for this, so nothing's likely to break if it's wrong yet.

Alternatively, you could append `include` to the path at line 9.